src/pages/workspaces/workspace/JobHistory.js

```diff
@@ -159,44 +160,34 @@ const JobHistory = _.flow(
           },
           columns: [
             {
-              size: { basis: 600, grow: 0 },
+              size: { basis: 500, grow: 0 },
               headerRenderer: () => h(HeaderCell, ['Job']),
               cellRenderer: ({ rowIndex }) => {
                 const {
-                  methodConfigurationNamespace, methodConfigurationName, submitter, submissionId, workflowStatuses,
-                  status, submissionEntity
+                  methodConfigurationNamespace, submitter, submissionId, workflowStatuses
                 } = submissions[rowIndex]
                 return h(Fragment, [
                   div([
                     div([
                       methodConfigurationNamespace !== namespace && span({ style: styles.deemphasized }, [
                         `${methodConfigurationNamespace}/`
                       ]),
-                      methodConfigurationName
+                      link({
+                        target: '_blank',
+                        //replace link with actual job manager link
+                        href: linkToJobManager ? `https://jobs.terra.bio/${namespace}/${name}/${submissionId}`: `${getConfig().firecloudUrlRoot}/#workspaces/${namespace}/${name}/monitor/${submissionId}`
+                      }, ['methodConfigurationName'])
```

Comment:
This is showing up in the UI as `methodConfigurationName`, rather than the value:
![screen shot 2019-02-04 at 4 07 55 pm](https://user-images.githubusercontent.com/2395211/52237253-0b85c680-2897-11e9-856b-1178dc6459dc.png)
It should be:
<img width="1054" alt="screen shot 2019-02-04 at 4 08 33 pm" src="https://user-images.githubusercontent.com/2395211/52237273-1c363c80-2897-11e9-92e6-c793346efdc0.png">

Reply:
wow big oops, thank you for pointing that out!!